### DISCUSSION TITLE
It was working fine and then after I added a coupel new loras and restarted it Errored out? Please Help!!

### DISCUSSION THREAD

**Discussion by PheonixAi420** [lllyasviel/Fooocus#3152] [Q&A] [answered]
![Screenshot 2024-06-16 113049](https://github.com/lllyasviel/Fooocus/assets/170890859/84522333-bec0-483c-9b6d-1569a3d1a749)

**Comment by PheonixAi420** [comment]
![Screenshot 2024-06-16 114810](https://github.com/lllyasviel/Fooocus/assets/170890859/1e61cc02-4210-440e-9400-4734b7d8dfcb)

**Reply by Rogala** [reply]
You're probably using Lora not sdxl.

**Reply by PheonixAi420** [reply]
No I only use SDXL 1.0 loras. It was working fine and then I put a couple of my new loras I just trained on Civitai and restarted the app and it did that. I'm in the process of installing fresh again and hopefully that will fix whatever happened but not sure yet.

**Reply by mashb1t** [reply]
let me check this, might be a bug introduced in https://github.com/lllyasviel/Fooocus/commit/3e453501f702a7d419dd4b4a2067ab6a7daca495

**Reply by mashb1t** [reply, marked as answer]
Fixed in https://github.com/lllyasviel/Fooocus/pull/3154, please restart Fooocus. Sorry.

**Reply by PheonixAi420** [reply]
I just reinstalled it and had to transfer my checkpoints and loras but its working now. Still not sure what happened but its fixed. Thanks.

**Reply by mashb1t** [reply]
You didn't have to do that, the last changes of main were missing a method, which i've added and which were automatically pulled.

**Reply by PheonixAi420** [reply]
Well its already done now, I was just impatient wanting to test out a new lora. I'm glad I posted the issue though so it could be addressed.
